test/Makefile

```diff
@@ -2,7 +2,7 @@
 
 testint:
 	@LINODE_FIXTURE_MODE="play" \
-	LINODE_TOKEN="awesometokenawesometokenawesometoken" \
+	LINODE_TOKEN="9f08d4534157252421c964c7b9bf53dd545a242c1f7ef4fe1ada1701366acf7f" \
```

Comment:
Accidental change?

Reply:
Yeah need to remove it

Reply:
This token is no longer valid. It has been revoked.